### PR TITLE
UITEN-248: Improve "Number of locations" on Campuses page

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
## Purpose

[UITEN-248](https://issues.folio.org/browse/UITEN-248)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by vashjs — APPROVED** [review]

**Review by Terala-Priyanka — COMMENTED** [review]
Could you add a screen cast?

File: src/settings/LocationCampuses.js [hunk 1]
```diff
@@ -84,23 +91,43 @@ class LocationCampuses extends React.Component {
...
+        pathname: 'location-libraries'
+      });
+
+      sessionStorage.setItem('institutionIdLibraries', this.state.institutionId);
+      sessionStorage.setItem('campusIdLibraries', item.id);
+    };
+
+    return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
+      <div
```
[the remark this answers is not in the document]
Comment (UladzislauKutarkin) [reply src/settings/LocationCampuses.js:117]: Because I don't think it's good idea to put button in cell inside of MLC.

**Comment by UladzislauKutarkin** [comment]
> Could you add a screen cast?

Yes. Screen Recording 2023-06-26 at 07.33.06.mov

File: src/settings/LocationCampuses.js [hunk 2]
```diff
@@ -84,23 +91,43 @@ class LocationCampuses extends React.Component {
...
+      this.props.history.replace({
+        pathname: 'location-libraries'
+      });
+
+      sessionStorage.setItem('institutionIdLibraries', this.state.institutionId);
+      sessionStorage.setItem('campusIdLibraries', item.id);
+    };
+
+    return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
```
Comment (NikitaSedyx) [inline src/settings/LocationCampuses.js:116]: it's invalid, our application should be accessible

or you make div accessible, or you use button

Comment (NikitaSedyx) [reply src/settings/LocationCampuses.js:116]: in fact it should be a link, not a button because your callback changes pathname

Comment (Terala-Priyanka) [reply src/settings/LocationCampuses.js:116]: I agree this has to be a <Link> component

**Review by Terala-Priyanka — CHANGES_REQUESTED** [review]

File: src/settings/LocationCampuses.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (BogdanDenis) [reply src/settings/LocationCampuses.js]: Maybe you could use `<TextLink>` from `stripes-components`. I think it's ok to use `<a>` inside cells

[the remark this answers is not in the document]
Comment (UladzislauKutarkin) [reply src/settings/LocationCampuses.js]: Done!

File: src/settings/LocationCampuses.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/settings/LocationCampuses.js]: Thanks @NikitaSedyx. I used `TextLink` in that case, and also made changes in previous place.

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**